### ISSUE TITLE
Docs for wp.data should have more links to Redux docs.

### ISSUE THREAD

**Issue by Shelob9** [WordPress/gutenberg#4452]
Because I'm not familiar with Redux, I'm finding it hard to following this page in the documentation:
https://github.com/WordPress/gutenberg/blob/master/data/README.md

It mentions a lot of Redux-specific concepts. So I'm googling them, and adding the links to the docs to make this easier on the next person to read the page.

**Comment by Shelob9** [comment]
PR #4453 includes a link to all of the pages I opened reading this page.

**Comment by danielbachhuber** [comment]
Looks like this landed in #4453
